backend/typescript/services/implementations/emailService.ts

```diff
@@ -1,7 +1,14 @@
 import nodemailer, { Transporter } from "nodemailer";
+import dayjs from "dayjs";
 import IEmailService from "../interfaces/emailService";
 import { NodemailerConfig } from "../../types";
 import logger from "../../utilities/logger";
+import Schedule from "../../models/scheduling.model";
+import User from "../../models/user.model";
+import createReminderEmailContent from "../../utilities/emailUtils";
+
+// eslint-disable-next-line
+const cron = require("node-cron");
```

Comment:
```suggestion
import * as cron from 'node-cron';
```
will this work? if not then just leave as is

might need `yarn add @types/node-cron`